src/app/technical-record-create/technical-record-create.component.scss

```diff
@@ -1,22 +1,4 @@
-@import "node_modules/govuk-frontend/govuk/all";
-
-.create-tech-record {
-  & h1:first-child {
-    margin-bottom: 1rem;
-  }
-  & .govuk-grid-row {
-    margin-bottom: 1.5rem;
-  }
-
-  & a#test-continue-btn {
-    padding: 10px 15px 10px 15px;
-    font-size: 20px;
-  }
-
-  & .govuk-error-summary {
-    padding-bottom: 0 !important;
-    & .govuk-error-message {
-      text-decoration: underline;
-    }
-  }
+.govuk-back-link {
```

Comment:
This solution should not be repeated in every file where the back link is needed